index.js

```diff
@@ -12,23 +12,31 @@ start();
 async function start() {
   let min = 1;
   let max = 100;
+  let guessArray = []
   let guess = Math.floor((min + max) / 2);
   console.log(
     "Let's play a game. Think of a number between 1-100 and I will try to guess it.\nNow I want you to let me know if I am higher or lower than your number when I guess it.\nPlease respond with H for higher,\nand L for lower for me to understand, Thanks!\nOK now lets begin!"
   );
+
   let hiLow = await ask("Is your number " + guess + "? H/L or Yes?");
   console.log(hiLow);
 
   while (hiLow.toUpperCase() !== "YES") {
-    if (hiLow.toUpperCase() === "L") {
+    if (max <= min){
```

Comment:
Off by one error. Which we have discussed, and I believe you've already fixed in a later commit